custom_components/evohome_cc/__init__.py

```diff
@@ -18,16 +18,26 @@
 except (ImportError, ModuleNotFoundError):
     import evohome_rf
 
+from evohome_rf.const import (
+    SYSTEM_MODE_MAP,
+    SYSTEM_MODE_LOOKUP 
```

Comment:
These consts should come from **evohome_cc.const** or **evohome_rf.const**, but not both, see below.

Reply:
I think I'll refactor to come from **evohome_rf**.